File: Facebook.NET/Response.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public class Response<T>
        {
            [DataMember(Name = "error")]
            public GraphApi.Error Error { get; set; }

            [DataMember(Name = "data")]
            public T Data { get; set; }
        }
    }
}

File: Facebook.NET - VB/Response.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        Private Sub New()
        End Sub
        
        <DataContract()>
        Public Class Response(Of T)
            <DataMember(Name:="error")>
            Public Property [Error]() As GraphApi.Error
            <DataMember(Name:="data")>
            Public Property Data() As T
        End Class
    End Class
End Namespace